ui/pages/swaps/prepare-swap-page/review-quote.js

```diff
@@ -143,7 +143,7 @@ import SlippageNotificationModal from './slippage-notification-modal';
 
 let intervalId;
 
-const GAS_FEES_LEARN_MORE_URL =
+export const GAS_FEES_LEARN_MORE_URL =
```

Comment:
Looks like this constant is hardcoded into multiple swaps files. Can you add it to [this file](https://github.com/MetaMask/metamask-extension/blob/b88cde41a7f3016733d1c4314507b77dd6adc26a/shared/lib/ui-utils.js) and replace usages in swaps/send instead?

Reply:
Yeah sure